src/wx/opts.cpp

```diff
@@ -277,6 +277,7 @@ opt_desc opts[] = {
 
     /// Geometry
     INTOPT("geometry/fullScreen", "Fullscreen", wxTRANSLATE("Enter fullscreen mode at startup"), fullScreen, 0, 1),
+    INTOPT("geometry/isMaximized", "Maximized", wxTRANSLATE("Window maximized"), windowMaximized, 0, 1),
```

Comment:
How do we handle the new option? Does it need to be translated somewhere? I believe this should not appear anywhere but in `vbam.ini`.

Reply:
You could also add it to `ConfigManager.{cpp,h}` but it's fine if it's not.